desktop/package.json

```diff
@@ -43,7 +43,7 @@
 		"enzyme": "^3.11.0",
 		"jest": "^27.2.4",
 		"lodash": "^4.17.21",
-		"playwright": "1.14.0",
+		"playwright": "1.16",
```

Comment:
Can we use `^` and use the major, minor, and patch numbers? To keep consistency with other version ranges.

Reply:
Sure, will use `^1.16.3`.

As an aside, I'm hoping the next push with `yarn dedupe` will fix the errors. E2E tests are failing inexplicably with updated version and it doesn't do so locally.